packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -450,4 +449,64 @@ table.kt-table {
 .hidden-columns {
 	display: none;
 }
+
+.kt-table ::v-deep .form-checkbox {
```

Comment:
is this a rebase mistake 🤔

Reply:
yes it was

the commit with which we seem to have conflicted is this
https://github.com/3YOURMIND/kotti/commit/11e06831d543fcd7cc187ebe2977c37cf280ff1c